skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -460,66 +446,60 @@ public function executeEditPage() {
 		}
 
 		$this->displaytitle = true;
-		$this->title = wfMsg( $titleMsg, htmlspecialchars( $wgTitle->getPrefixedText() ) );
+		$this->title = wfMessage( $titleMsg, $this->wg->Title->getPrefixedText() )->parse();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)